docs/content/en/docs/operator-manual/piped/configuration-reference.md

```diff
@@ -170,8 +170,8 @@ Must be one of the following structs:
 |-|-|-|-|
 | repoId | string | Id of the git repository. This must be unique within the repos' elements. | Yes |
 | commitMessage | string | The commit message used to push after replacing values. Default message is used if not given. | No |
-| includes | []string | The paths to EventWatcher files to be included. | No |
-| excludes | []string | The paths to EventWatcher files to be excluded. This is prioritized if both includes and this are given. | No |
+| includes | []string | The paths to EventWatcher files to be included. Patterns can be used like `foo/*.yaml`. | No |
+| excludes | []string | The paths to EventWatcher files to be excluded. Patterns can be used like `foo/*.yaml`. This is prioritized if both includes and this are given. | No |
 
```

Comment:
Out of this PR scope, but I started thinking that instead of those `includes`, `excludes` and `.pipe` directory we can provide a `path` field and it can be a path to a single file or a directory to load EventWatcher configuration. Easier to understand, configure, and don't have to care about the relation with the `.pipe` directory.

Reply:
Like for instance, 

```yaml
apiVersion: pipecd.dev/v1beta1
kind: Piped
spec:
  eventWatcher:
    gitRepos:
      - repoId: repo-1
        path: foo/bar # or foo/bar/event-watcher.yaml
```

Is it what you mean?

Reply:
Yes, more precisely

``` yaml
apiVersion: pipecd.dev/v1beta1
kind: Piped
spec:
  eventWatcher:
    gitRepos:
      - repoId: repo-1
        path: .piped/foo   # or .piped/event-watcher.yaml
```

Reply:
Looks neat 👍 

But it defaults to `.pipe` when not given, right?

Reply:
To be obvious, I think that the `path` field should be a relative path from the root of the repository.
So do not care about `.pipe` in this case. Just use whatever the user configured.
(To be honest, we also want to stop being dependent on the `.pipe` directory.)

Reply:
In that case, users have to determine what directory should be used and specify it for each config (as well as analysis template). To offer default directory is still good for those who want to use it in the general case.

Reply:
I would say being able to specify other directories is a great choice definitely. However, I don't want to burden users who don't need it any more than we already do.

Reply:
I see. I think adding more default values like that just makes a new thing to remember.
Besides that we are having `pipe`, `piped`, `pipecd`, they are easy to confuse.

Reply:
I hear you, let's talk about it in another thread 👍

Reply:
👍